packages/asc-ui/src/components/CompactPager/CompactPager.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import { ChevronRight, ChevronLeft } from '@datapunt/asc-assets'
+import CompactPagerStyle, { PagerText, PagerButton } from './CompactPagerStyle'
+
+function clampNumber(value: number, min: number, max: number) {
+  return Math.min(Math.max(value, min), max)
+}
+
+export function determineTotalPages(collectionSize: number, pageSize: number) {
```

Comment:
Can't we use the props as defined below for `collectionSize` and `pagesize`? Maybe move these two functions two a `utils.ts` and define the props there

Reply:
Not quite sure how to optimize this to be honest. I think it's fine for these functions to live next to the component like this since they are exclusively used there.

We could pass the entire props to this method, but then we also pass a lot of information that this function doesn't need, also complicating the tests more.